### DISCUSSION TITLE
image preview not work in kitty with zellij on ?

### DISCUSSION THREAD

**Discussion by OriLiMu** [sxyazi/yazi#1176] [1. Q&A] [answered]
zellij kitty yazi all get latest version at this moment . wezterm works with zellij on.

**Comment by sxyazi** [comment, marked as answer]
Zellij doesn't support Kitty, see https://github.com/zellij-org/zellij/issues/2814

**Reply by OriLiMu** [reply]
Hi I just install the ueberzugpp instead of ueberzug the image preview works on zellij in kitty . It is the issue just solved ?
